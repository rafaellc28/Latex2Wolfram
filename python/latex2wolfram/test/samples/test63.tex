% Test cosecant expression
\csc(1.2+x)
