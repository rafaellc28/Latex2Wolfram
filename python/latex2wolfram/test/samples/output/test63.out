csc(1.2 + x)
